src/game/level.h

```diff
@@ -14,17 +15,53 @@
 #include <string>
 #include <vector>
 
+struct Background {
+  i64 x;
+  i64 y;
+  std::vector<u32> background_data;
+};
+
+struct Objects {
+  i64 x;
```

Comment:
position (`x` and `y`) should probably be `f32`, converted from Tiled coordinate system (pixels) to our in-engine coordinate system (meters)